x-pack/solutions/chat/plugins/workchat-app/server/saved_objects/agents.ts

```diff
@@ -24,6 +24,12 @@ export const agentSoType: SavedObjectsType<AgentAttributes> = {
       configuration: { dynamic: false, type: 'object', properties: {} },
       user_id: { type: 'keyword' },
       user_name: { type: 'keyword' },
+      avatar: {
```

Comment:
The correct and safe way to add fields is to introduce a new model version with a migration that [adds mappings](https://github.com/elastic/kibana/blob/1575f51019ce37de5c704f598eae02b9f95842ce/src/core/packages/saved-objects/server/src/model_version/model_change.ts#L56-L68)

Reply:
Are you intending to query against avatar color and text? Unless you have a use case for querying these fields you should rather not index them and instead just use `dynamic: true`

See the paragraph starting with "Do not use field mappings like you would use data types for the columns of a SQL database."
https://github.com/elastic/kibana/blob/main/docs/extend/saved-objects-service.md#mappings-_mappings